### PR TITLE
Add install GitHub Action

### PR DESCRIPTION
**Pull request by bryanoltman**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: 'Install Shorebird'
+description: 'Installs shorebird'
+runs:
+  using: "composite"
+  steps:
+    - id: install
```
Comment (felangel) [inline action.yml:6]: Since shorebird requires Flutter and Dart should we also install those as steps?

Comment (bryanoltman) [reply action.yml:6]: Good idea, will add

File: action.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
```
Comment (felangel) [inline action.yml:6]: Also nit but if we're not using the `id` for anything we can probably omit it

File: action.yml (hunk 1 above)
Comment (bryanoltman) [reply action.yml]: Done.

File: action.yml (hunk 2 above)
Comment (bryanoltman) [reply action.yml]: Done.

File: action.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+name: 'Install Shorebird'
+description: 'Installs shorebird'
+runs:
+  using: "composite"
+  steps:
+    - name: install-flutter
```
Comment (felangel) [inline action.yml:6]:
```suggestion
    - name: Install Flutter
```
nit for consistency

**Pull request merged**